samples/samples.dpr

```diff
@@ -6,10 +6,19 @@ uses
   Horse,
   Horse.OctetStream,
   System.Classes,
-  System.SysUtils;
+  System.SysUtils,
+  Winapi.Windows;
```

Comment:
ExtractFilePath does not need uses Winapi.Windows

Reply:
Yes, sorry I forgot that the code refers to WinApi. I believe that the only case in which ParamStr (0) will not work is ISAPI.